charts/model-engine/templates/inference_framework_config.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: model-engine-inference-framework-latest-config
```

Comment:
shouldn't this configmap be in the same namespace with all the other stuff?

Reply:
it should be in the same namespace as the gateway, so default

Reply:
with sgp installs we're not guaranteed that it's in the default namespace

Reply:
i think you can record model engine namespace in service config yaml as
```
model_engine_namespace: {{.Release.Namespace}}
```